README.md

```diff
@@ -18,6 +18,53 @@ We maintain a [list](api-go/config/multi-repo.md) which you could directly add y
 
 [![Monthly Active Contributors](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorMonthlyActivity&repo=apache/apisix&merge=true)](https://www.apiseven.com/en/contributor-graph?chart=contributorMonthlyActivity&repo=apache/apisix&merge=true)
 
+## Development
```

Comment:
Since normal contributors could not get access by default, so maybe we could move it to a separate file and leave a link here, so not all people need to see it on README.

Also maybe we could also add guidance on how to get access.

Reply:
Agree, we do need a guide to tell a new developer how to start this project.

Reply:
Yes it's pretty complicated, since we deploy it on something we pay for, we need to find the way to give enough permission for users to contribute, but also avoid people abusing it